meetings/2021-03-08.md

```diff
@@ -0,0 +1,44 @@
+# 2021-03-08 Authentication Panel
+
+## Meeting
+ * Call: https://meet.jit.si/solid-authentication
+
+## Present
+
+* Henry Story
+* Sarven Capadisli
+* Elf Pavlik
+* Matthieu Bosquet
+* Aaron Coburn
+
+## Agenda
+
+* [Adding meeting notes from 2021-03-01](https://github.com/solid/authentication-panel/pull/134) -- (2 minutes)
+*  [Link to Http Signature doc](https://github.com/solid/authentication-panel/pull/140) PR (5 minutes)
+* [Audience Claim Issue](https://github.com/solid/authentication-panel/issues/135) & [related PR](https://github.com/solid/authentication-panel/pull/141) (5 minutes)
+
+## Notes
+
+### HTTP Signature
+
+Henry: The link from the front page will allow me to ask Manu where the work on Http Signatures is going. In his recent talk on the wide ranging work of the W3C Credentials CG there is an entry of HTTP Signatures. See  https://twitter.com/bblfish/status/1368897757365288965
+
+### Audience Claim
+
+Henry: Actually I think that working with existing identity providers is the main use case for OIDC.
+
+Aaron: Should we have a particular scope for requesting access tokens? Using the "solid" scope, the IdP would know to include agent and client WebID...
+
+Sarven: Would scope be the best way to request a solid access token?
+
+Aaron: It would require a bit of work on the client side.
+
+Henry: Could one not say that it is not Solid dependent to authenticate but really only WebID dependent, since all one needs is to publish a WebID document on an simple Web (e.g. Apache) server.
```

Comment:
```suggestion
Henry: Could one not say that it is not Solid dependent to authenticate but really only WebID dependent, since all one needs is to publish a WebID document on a simple Web server (e.g., Apache).
```